controller/tests/support/config.py

```diff
@@ -36,6 +39,10 @@ async def configure(
     Config
         New configuration.
     """
+    monkeypatch.setenv("METRICS_APPLICATION", "controller")
+    monkeypatch.setenv("METRICS_ENABLED", "false")
+    monkeypatch.setenv("METRICS_MOCK", "true")
+
```

Comment:
Could this be in an autouse fixture so you don't have to thread a `monkeypatch` fixture through all the tests?

Reply:
Good call, yes. Fixing, thanks!